io.openems.edge.ess.sma/test/io/openems/edge/sma/sunnyisland/EssSmaSunnyIslandImplTest.java

```diff
@@ -21,7 +23,9 @@ public void test() throws Exception {
 						.setPhase(L1) //
 						.setCapacity(10_000) //
 						.build()) //
-		;
+				.next(new TestCase() //
+						.output(SymmetricEss.ChannelId.CAPACITY, 10_000)) //
+				.deactivate();
```

Comment:
Thanks @sfeilmeier! It should have been possible for me, to add this by myself ;-)